Anshul/ESPN.PY

```diff
@@ -0,0 +1,63 @@
+import time
+
+from selenium import webdriver
+
+import unittest
+
+from selenium.webdriver import ActionChains
+from selenium.webdriver.common.by import By
+
+
+class ESPN(unittest.TestCase):
+    def setUp(self) -> None:
+        global IPL1
+        IPL1=webdriver.Chrome()
+        IPL1.get("https://www.espncricinfo.com")
+        IPL1.maximize_window()
+
+
+    def test_OutPlyerNmae(self):
+        KeySeriessexction=IPL1.find_element(By.XPATH,"//span[text()='World Cup Super League']")
+        Ac=ActionChains(IPL1)
+        Ac.move_to_element(KeySeriessexction).perform()
+        IPL2023=IPL1.find_element(By.XPATH,"//span[text()='IPL 2023']/parent::a")
+        IPL2023.click()
+        time.sleep(20)
```

Comment:
Can we replace this hardcoded wait with explicit wait ?
Have we tried applying explicit wait ?
Please check if explicit wait works here.